.github/workflows/continuous-integration.yaml

```diff
@@ -562,22 +564,17 @@ jobs:
           && steps.determinism.outcome != 'failure' 
           && !cancelled()
         run: |
-          make docs
-
-      # The changelog is generated by considering all commits from the latest stable previous 
-      # version (not a release candidate) up to the new upcoming version
-      # This may change in the future
-      # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3909
-      - name: Generate release changelog
-        id: changelog
-        if: ${{ fromJSON(env.IS_RELEASE) && steps.install-deps.outcome == 'success' && !cancelled() }}
+          make docs_no_links
+      
+      # Do not check links during the release process in order to avoid temporary connection errors
+      - name: Check links
+        id: check_links
+        if: |
+          !fromJSON(env.IS_RELEASE)
```

Comment:
only check links when we are not releasing

Reply:
That's great!